components/infobox/commons/infobox_widget_factory.lua

```diff
@@ -8,10 +8,16 @@
 
 local Class = require('Module:Class')
 local Customizable = require('Module:Infobox/Widget/Customizable')
+local String = require('Module:StringUtils')
 local Widget = require('Module:Infobox/Widget')
 
 local WidgetFactory = Class.new()
 
+local _ERROR_TEXT = '<span style="color:#ff0000;font-weight:bold" class="show-when-logged-in">' ..
+					'Unexpected Error, report this in #bugs on our [https://discord.gg/liquipedia Discord]. ' ..
+					'${errorMessage}' ..
+					'</span>[[Category:Pages with script errors]]'
```

Comment:
could maybe be useful to have that in a sep module so we can access it from other modules too